sla/sla-core/pom.xml

```diff
@@ -113,17 +114,17 @@
 
 	<dependencies>
 		<dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>${junit.version}</version>
-            <scope>test</scope>
+			<groupId>junit</groupId>
```

Comment:
The TestNG eclipse plugin converts JUnit tests to TestNG tests, but the ones run with SpringRunner. The process for these tests is not easy and has problems (http://stackoverflow.com/questions/2608528/spring-dependency-injection-with-testng, http://stackoverflow.com/questions/5192562/spring-autowiring-happens-after-beforeclass-when-running-test-with-maven-surefi).

Having both dependencies, the effect is that TestNG is able to run the JUnit tests (http://testng.org/doc/migrating.html).